microcosm_pubsub/producer.py

```diff
@@ -43,10 +59,10 @@ def produce(self, media_type, dct=None, uri=None, **kwargs):
 
         if self.skip:
             return
-        message, topic_arn, opaque_data = self.create_message(media_type, dct, uri, **kwargs)
-        return self.publish_message(media_type, message, topic_arn, opaque_data)
+        pubsub_message = self.create_message(media_type, dct, uri, **kwargs)
```

Comment:
This list was getting longer, so I just put all of the needed fields into a container.